source/PyMaterialX/PyMaterialXCore/PyElement.cpp

```diff
@@ -26,9 +26,36 @@ namespace mx = MaterialX;
 
 void bindPyElement(py::module& mod)
 {
+    py::class_<mx::ElementEquivalenceResult>(mod, "ElementEquivalenceResult")
+        .def_readonly_static("ATTRIBUTE", &mx::ElementEquivalenceResult::ATTRIBUTE)
+        .def_readonly_static("ATTRIBUTE_NAMES", &mx::ElementEquivalenceResult::ATTRIBUTE_NAMES)
+        .def_readonly_static("CHILD_COUNT", &mx::ElementEquivalenceResult::CHILD_COUNT)
+        .def_readonly_static("CHILD_NAME", &mx::ElementEquivalenceResult::CHILD_NAME)
+        .def_readonly_static("NAME", &mx::ElementEquivalenceResult::NAME)
+        .def_readonly_static("CATEGORY", &mx::ElementEquivalenceResult::CATEGORY)
+        .def_readwrite("path1", &mx::ElementEquivalenceResult::path1)
+        .def_readwrite("path2", &mx::ElementEquivalenceResult::path2)
+        .def_readwrite("differenceType", &mx::ElementEquivalenceResult::differenceType)
+        .def_readwrite("attributeName", &mx::ElementEquivalenceResult::attributeName);
+
+    py::class_<mx::ElementEquivalenceOptions>(mod, "ElementEquivalenceOptions")
+        .def_readwrite("format", &mx::ElementEquivalenceOptions::format)
+        .def_readwrite("precision", &mx::ElementEquivalenceOptions::precision)
+        .def_readwrite("skipAttributes", &mx::ElementEquivalenceOptions::skipAttributes)
+        .def_readwrite("skipValueComparisons", &mx::ElementEquivalenceOptions::skipValueComparisons)
+        .def(py::init<>());
+
     py::class_<mx::Element, mx::ElementPtr>(mod, "Element")
         .def(py::self == py::self)
         .def(py::self != py::self)
+        .def("isEquivalent", [](const mx::Element& elem, mx::ConstElementPtr& rhs, const mx::ElementEquivalenceOptions& options)
+        {
+            mx::ElementEquivalenceResultVec results;
+            bool res = elem.isEquivalent(rhs, options, &results);
+            return std::pair<bool, mx::ElementEquivalenceResultVec>(res, results);
+        })        
+        .def("", &mx::Element::isEquivalent,
```

Comment:
What is the motivation for providing _two_ interfaces for this method, rather than using the unified approach seen below in `Element::validate`?  Would it make sense to adopt the `validate` approach here as well?

Reply:
I think I had a bad merge. This second declaration should be deleted.